.release/ami_install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+
+DIRECTORY=`dirname $0`
+source $DIRECTORY/bootstrap_steps.sh
+
+echo "LOADED UP BOOTSTRAP STEPS"
+
+install_wallaroo() {
+  PYTHON_PATH=$(python -c "import site; print(site.getsitepackages()[0])")
+  git checkout "$WALLAROO_VERSION"
+  sudo cp machida/wallaroo.py "$PYTHON_PATH"
+  (make build-machida && sudo sudo cp machida/build/machida /usr/bin)
```

Comment:
This (and the following few `make` commands) will by default compile `machida` (and the other apps) optimized for the processor they're being compiled on. It is probably a good idea to compile for generic X86-64 for maximum compatibility. We can probably pick a non-generic/newer baseline intel processor to compile for to ensure better performance and no `illegal instruction` issues.

Reply:
That makes total sense @dipinhora. Are the specific flags you recommend setting documented somewhere? I can look into this if you point me in the right direction.

Reply:
Use `make target_cpu=<FANCYCPUNAME>` to target a specific cpu. According to https://aws.amazon.com/intel/, it looks like `Sandy Bridge` is still in use on some instance types so that's likely what we should target. The following should work: `make target_cpu=sandybridge`.